system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -688,6 +690,7 @@ class CLdapConnection : implements ILdapConnection, public CInterface
         }
     }
 
+private:
     virtual int connect(const char* ldapserver, const char* protocol)
```

Comment:
I built with LN branch successfully.  But you do make a good general point about changing the interface on any externally facing objects. Good topic to discuss in another Jira?